Dockerfile

```
@@ -1,8 +1,26 @@
-FROM node:9 as build-deps
+FROM node:10-alpine
```

`npm install` will fail on Node.js > v9. So we should use v9

```suggestion
FROM node:9-alpine
```